packages/indexer-common/src/allocations/query-fees.ts

```diff
@@ -624,6 +826,126 @@ export class AllocationReceiptCollector implements ReceiptCollector {
     }
   }
 
+  private async submitRAVs(signedRavs: RavWithAllocation[]): Promise<void> {
+    const logger = this.logger.child({
+      function: 'submitRAVs()',
+      signedRavSize: signedRavs.length,
+    })
+    if (!this.tapContracts) {
+      logger.error(
+        `Undefined escrow contracts, but this shouldn't happen as RAV process is only triggered when escrow is provided`,
+        {
+          signedRavs,
+        },
+      )
+      return
+    }
+    const escrow = this.tapContracts
+
+    logger.info(`Redeem last RAVs on chain individually`, {
+      signedRavs,
+    })
+
+    // Redeem RAV one-by-one as no plual version available
+    for (const { rav: signedRav, allocation, sender } of signedRavs) {
+      const { rav } = signedRav
+      const stopTimer = this.metrics.ravsRedeemDuration.startTimer({
+        allocation: rav.allocationId,
+      })
+      try {
+        // compute allocation id proof
+        //
```

Comment:
Format and cleanup comment?

Reply:
This was also changed :
https://github.com/graphprotocol/indexer/blob/gusinacio/collect-tap-ravs/packages/indexer-common/src/allocations/query-fees.ts#L904-L918